config/application.rb

```diff
@@ -24,6 +24,9 @@ class Application < Rails::Application
       generate.controller_specs false
     end
 
+    config.i18n.available_locales = :en, :sv
```

Comment:
English is the only language available. Why is 'sv' included?

Reply:
It is not necessary. Just to show that it is possible as a available language. It doesn't effect anything. 
We can remove it.

Reply:
Removed